blocks/hooks/index.js

```diff
@@ -11,29 +6,7 @@ import customClassName from './custom-class-name';
 import generatedClassName from './generated-class-name';
 import matchers from './matchers';
 
-const hooks = createHooks();
-
-const {
-	addFilter,
-	removeFilter,
-	removeAllFilters,
-	applyFilters,
-	doingFilter,
-	didFilter,
-	hasFilter,
-} = hooks;
-
-export {
-	addFilter,
-	removeFilter,
-	removeAllFilters,
-	applyFilters,
-	doingFilter,
-	didFilter,
-	hasFilter,
-};
-
-anchor( hooks );
-customClassName( hooks );
-generatedClassName( hooks );
-matchers( hooks );
+anchor();
```

Comment:
Minor: Do we need to export these as functions anymore, or can they just call to `addFilter` when imported?

Reply:
It might not play well with unit tests. Need to verify.

Reply:
In theory, we could export individual functions and apply filters when importing. I will open a follow-up so we could discuss it individually.